example/android/app/src/main/jni/Android.mk

```diff
@@ -5,6 +5,10 @@ include $(REACT_ANDROID_DIR)/Android-prebuilt.mk
 # If you wish to add a custom TurboModule or Fabric component in your app you
 # will have to include the following autogenerated makefile.
 # include $(GENERATED_SRC_DIR)/codegen/jni/Android.mk
+
+# Includes the MK file for `react-native-fast-shadow`
+include $(NODE_MODULES_DIR)/../../android/build/generated/source/codegen/jni/Android.mk
```

Comment:
it's pointed to android folder directly due to example app folder structure, on end user app it should be like this `$(NODE_MODULES_DIR)/react-native-fast-shadow/android/build/generated/source/codegen/jni/Android.mk`